### PR TITLE
Updated migration list table columns width

### PR DESCRIPTION
**Pull request by akshika898**
Updated width of table columns without content truncation
Linked issue: https://github.com/microsoft/azuredatastudio/issues/25914

https://github.com/user-attachments/assets/93adb3a3-4180-476c-a45d-516298719d5d

### CODE REVIEW

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -1,96 +1,46 @@
...
-    branches: [ "main", "extensions/rc1", "master", "release/*", "release/extensions" ]
   schedule:
-    - cron: '35 3 * * 1'
-  workflow_dispatch:
+    - cron: '0 0 * * 0'
 
 jobs:
-  analyze:
-    name: Analyze (${{ matrix.language }})
-    # Runner size impacts CodeQL analysis time. To learn more, please see:
```
Comment (sahuroshan) [inline .github/workflows/codeql.yml:26]: Why this change on codeql file? Is it required ?

Comment (akshika898) [reply .github/workflows/codeql.yml]: Not required, reverted.

**Review by sahuroshan — APPROVED** [review]
